### PR TITLE
Add help menu

### PR DESCRIPTION
**Pull request by LWachtel1**
Add HelpMenuManager, update ByteCeps & UserInterface accordingly

### CODE REVIEW

**Review by pqienso — APPROVED** [review]
LGTM, thanks!

File: src/main/java/byteceps/processing/HelpMenuManager.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+            for (int i = 0; i < exerciseFlagFunctions.length; i++) {
+                result.append(String.format("\t\t\t %s%s", getExerciseFlagFunctions(i), System.lineSeparator()));
+
+            }
+
+            String exerciseCommandChoices = result.toString();
+
+            return exerciseCommandChoices;
+
+        } else {
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/processing/HelpMenuManager.java:140]: Early returns don't need an else (for the other methods as well)

File: src/main/java/byteceps/processing/HelpMenuManager.java [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        default:
+            throw new Exceptions.InvalidInput("Command type is not recognised");
+        }
+
+        return menuSelection;
+    }
+
+
+    public String showExerciseCommand(Parser parser) {
+
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/processing/HelpMenuManager.java:120]: Able to remove these white spaces? A few new lines that should be condensed.

**Review by joshualeejunyi — COMMENTED** [review]
No glaring issues, can merge either way, just thought could clean up a bit more.

**Pull request merged**